### PR TITLE
[DUOS-1668][risk=low] Split out signing official console into 3 separate routes.

### PR DESCRIPTION
**Pull request by lu-c**
Added 3 new routes for each of the signing official console tabs, copy pasted the signing official console page and deleted all irrelevant code to the tab

- To test, enable the duos header in the javascript console:
`localStorage.setItem('feature-flag:navstyle', 'DUOS-1668')`

### CODE REVIEW

**Comment by rushtong** [comment]
Another ticket I think we need is to standardize on title language, So, not for this PR, but we should add another ticket to clear this up with Jonathan/Yvonne. For example, we have:

Admin
* DAR Requests -> Manage Data Access Request Collections

Chair
* Manage DARs -> Manage Data Access Request Collections

Member
* Manage Data Access Request Collections

SO
* DAR Requests -> DAR Requests

Researchers
* DAR Collections

**Comment by rushtong** [comment]
The "Data Owner Console" has been removed, let's get rid of that link in the main nav.

**Review by rushtong — APPROVED** [review]
Code looks good! 👍🏽

**Comment by lu-c** [comment]
> The "Data Owner Console" has been removed, let's get rid of that link in the main nav.

Is this the same console in this ticket? https://broadworkbench.atlassian.net/browse/DUOS-1666 Just wanted to clarify (since I wanted to start on that one next)

**Comment by rushtong** [comment]
> > The "Data Owner Console" has been removed, let's get rid of that link in the main nav.
> 
> Is this the same console in this ticket? https://broadworkbench.atlassian.net/browse/DUOS-1666 Just wanted to clarify (since I wanted to start on that one next)

That ticket refers to the Data Access Committee console. There are two flavors of DAC user, the Chairperson and the Member

File: src/components/signing_official_table/SigningOfficialTable.js [hunk 1]
```diff
@@ -401,7 +401,15 @@ export default function SigningOfficialTable(props) {
...
-          },['My Institution\'s Researchers. Issue or Remove Researcher privileges below.']),
+          },[
+            'My Institution\'s Researchers. Issue or Remove Researcher privileges below. ',
+            a({
+              rel: 'noopener noreferrer',
+              href: 'https://broad-duos.zendesk.com/hc/en-us/articles/360060402751-Signing-Official-User-Guide',
+              target: '_blank',
+              id: 'so-console-info-link'},
+            ['Click Here for more information'])
+          ]),
```
Comment (shaemarks) [inline src/components/signing_official_table/SigningOfficialTable.js]: Nit: The spacing between the Search Bar and Add New Researcher Button has been lost due to the added text. Would it be possible to add some space back, possibly by putting the link on the following line? 

Add New Researcher Before
Add New Researcher After

**Review by shaemarks — COMMENTED** [review]
Code and functionality looks good to me! Small nit regarding spacing, but otherwise 👍

File: src/components/signing_official_table/SigningOfficialTable.js (hunk 1 above)
Comment (lu-c) [reply src/components/signing_official_table/SigningOfficialTable.js]: added a lil padding to this!

**Comment by rushtong** [comment]
Pinging @shaemarks - can you give an official 👍🏽 to this PR? Please and thank you!

**Review by shaemarks — APPROVED** [review]
Looks good! Thanks for the changes!

**Pull request merged**